packages/theme/components/AppFooter.vue

```diff
@@ -90,7 +90,7 @@
     </SfFooterColumn>
     <SfFooterColumn title="Social">
       <div class="footer__socials">
-        <SfImage class="footer__social-image" v-for="item in social" :key="item" :src="'/icons/'+item+'.svg'" :alt="item" width="32" height="32" />
+        <SfImage class="footer__social-image" v-for="item in social" :key="item" :src="'/icons/'+item+'.svg'" :alt="item" :width=32 :height=32 />
```

Comment:
Are these colon and semicolon changes required by the SFUI update? Because it seems more like a linter configuration

Reply:
As per the documentation,SfImage: width and height props types changed to number and both are required and SfCard, SfGallery, SfCollectedProduct, SfGroupedProductItem, SfProductCard, SfProductCardHorizontal: props types of imageWidth and imageHeight changed to Number.
If we don't add semicolon it considers them as string by default.